nservicebus/hosting/index.md

```diff
@@ -37,7 +37,7 @@ The user is also responsible for properly shutting down the endpoint when it is
 
 snippet:Hosting-Shutdown
 
-NOTE: Starting with V6 we no longer provide the `Dispose` method. This is because there is now way to flow the asynchronous context through the `Dispose` method. The shutdown procedure might involve executing async code (e.g. `IWantToRunWhenBusStartsAndStops`) which means that if `Dispose` would wrap `Stop` with a `GetResult()` call, it could cause deadlocks.
+NOTE: Starting with Version 6 we no longer provide the `Dispose` method. This is because there is now way to flow the asynchronous context through the `Dispose` method. The shutdown procedure might involve executing async code (e.g. `IWantToRunWhenBusStartsAndStops`) which means that if `Dispose` would wrap `Stop` with a `GetResult()` call, it could cause deadlocks.
```

Comment:
> which means that if `Dispose` would wrap `Stop` with a `GetResult()` call

I know that was there already, but I'd avoid `GetResult` since most users won't have heard of that before. Can we rephrase that to use `Wait()` and `Result` instead?